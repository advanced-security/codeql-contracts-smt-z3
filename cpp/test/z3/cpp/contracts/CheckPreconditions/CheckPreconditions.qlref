z3/cpp/contracts/CheckPreconditions.ql