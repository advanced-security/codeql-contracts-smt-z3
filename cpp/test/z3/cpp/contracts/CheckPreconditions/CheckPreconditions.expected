| main.cpp:12:5:12:6 | call to f1 | The $@ at guarding this function call may not be satisfied by this implementation. The SMT Solver found the following model which violates this method's precondition: \n\n(define-fun _valueType__b () Int\n  0)\n(define-fun _valueType__a () Int\n  0) | main.cpp:3:6:3:7 | specification of f1 | specification case |
| main.cpp:19:5:19:6 | call to f1 | The $@ at guarding this function call may not be satisfied by this implementation. The SMT Solver found the following model which violates this method's precondition: \n\n(define-fun _valueType__b () Int\n  10)\n(define-fun _valueType__a () Int\n  1) | main.cpp:3:6:3:7 | specification of f1 | specification case |
